### ISSUE TITLE
Deletion inconsistency in SIF edges

### ISSUE THREAD

**Issue by hasanbalci** [iVis-at-Bilkent/newt#474]
While topology grouping of SIF nodes is enabled, SIF edges cannot be deleted by using 'Delete' key or using 'Delete Selected Simple' in the Edit Menu. However, they can be deleted by using 'Delete' button in the context-menu of the associated edge.

**Comment by ugurdogrusoz** [comment]
@metincansiper do you have time to disable the shortcuts as well for this map type?

**Comment by metincansiper** [comment]
Delete was still working because the shortcut was not using ``chiseInstance.deleteElesSimple()``. I updated it and now remove does not work when topology grouping is applied. 

Since sif compounds have a selector I am able to tell context menus extension to consider "collapse" option only the compounds who does not have that selector. In this way I was able to not to show "collapse" option when a SIF map is open (since all the compounds in SIF map have the related selector).

Now I removed that extra check and "collapse" in context menu behaves like "delete" in context menu in that manner (for consistency reasons). I mean it is visible even if the topology grouping is enabled but nothing happens when clicked. If you wish I can revert the change in "collapse" back.
